Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,463 @@
+git push dokku HEAD:main
+```
+
+Dokku will then show deploy logs:
+
+```
+remote: Resolving deltas: 100% (5066/5066), done.
+-----> Set main as deploy-branch
+-----> Cleaning up...
+-----> Building platform from Dockerfile
+remote: #0 building with "default" instance using docker driver
+remote:
+remote: #1 [internal] load build definition from Dockerfile
+remote: #1 transferring dockerfile: 1.50kB done
+remote: #1 DONE 0.1s
+remote:
+remote: #2 [internal] load metadata for mcr.microsoft.com/dotnet/sdk:8.0
+remote: #2 DONE 0.2s
+remote:
+remote: #3 [internal] load metadata for docker.io/library/debian:latest
+remote: #3 DONE 0.4s
+remote:
+remote: #4 [internal] load .dockerignore
+remote: #4 transferring context: 51B done
+remote: #4 DONE 0.0s
+
+### Truncated for Brevity
+
+=====> Application deployed:
+       http://platform.jlyon1.catenatools.com
+       https://platform.jlyon1.catenatools.com
+
+To jlyon1.catenatools.com:platform
+ * [new branch]      HEAD -> main
+```
+
+Now we're all set! 
+
+> You can update your platform the same way, simply commit the changes and push them to the `main` branch on the dokku remote. The server will automatically build and deploy your changes with no downtime.
+
+
+You can test your platform by logging in:
+
+```Bash
+SESSION_ID=$(curl -D - -sS --location 'https://platform.jlyon1.catenatools.com/api/v1/authentication/login' \
+      --header 'Content-Type: application/json' \
+      --data '{
+      "provider": "PROVIDER_UNSAFE",
+      "payload": "test53"
+  }' | grep "session-*" | cut -d' ' -f2)
+
+curl --location 'https://platform.jlyon1.catenatools.com/api/v1/accounts' \
+--header "session-id: $SESSION_ID" \
+--header 'Content-Type: application/json' \
+--data '{}'
+
+# {"account":{"id":"account-f378e3c2-8760-4059-ba91-69f5343dfb48","displayName":"test53","authRole":"user","metadata":{},"providers":["PROVIDER_UNSAFE"]}}
+```
+
+
+The full postman collection can be found here. Set the http-host environment variable to your domain prior to use.
+
+## Running your game in a multiplayer configuration
+
+Next we will use the [Sidecar](Sidecar.md) along with our backend to support a multiplayer workflow.
+
+We are making some assumptions and trade-offs to simplify this guide. There is an example implementation in our [catena-lyra-demo repo.](https://github.com/CatenaTools/catena-lyra-demo/tree/main)
+
+1. The server will be "always open" and accepting players
+2. There will be only one type of match
+
+
+### Setup our Game Server
+
+We will be running our game on an AWS EC2 Instance running Windows Server, these instructions also work with a Linux sever using minor modifications.
+
+First create a key pair in the AWS console for remote access to the machine:
+
+![access_key_quickstart](access_key_quickstart.png)
+
+Next, configure terraform to provision an ec2 instance where we will run the server, this should be in `ec2/main.tf`
+
+```
+resource aws_instance "server_instance" {
```

Comment:
Should I just make another top level folder next to `catena-core/` in the `aws/` folder called `windows-game-server/` that implements all this so the end user just has to run the terraform commands?